ansible-roles/os/ubuntu/clevis-install.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install clevis packages
+  hosts: qemu_ubuntu
```

Comment:
@DaniilKl 

`become: true` is missing here

Reply:
This playbook has been deleted due to the fact that we need `clevis` only on Ubuntu with encrypted `rootfs`. `clevis` is being installed now only in test suite.